doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -5319,7 +5319,7 @@ \subsubsection{Outputs}\label{outputs-13-003}
 
 \subsection{HeatPump:PlantLoop:EIR:Cooling (and Heating)}\label{plhp_eir}
 
```

Comment:
The Cooling and Heating versions are diverging inputs with these changes, for example heating version has defrost stuff.  I think this I/O reference entry will need to be split up into two in order to handle the input descriptions. cleanly.  

And more description of new air-source capababilities.